packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/bodyComplexRestClient.ts

```diff
@@ -31,5 +30,6 @@ export default function createClient(
 
   const client = getClient(baseUrl, options) as BodyComplexRestClient;
 
+  client.pipeline.removePolicy({ name: "ApiVersionPolicy" });
```

Comment:
A little bit suprised with this change, does that mean there is a path-level apiVersion in client?

Reply:
No, this means, there's no query api version in client, there're three cases, we don't need this policy.
1. client level path parameter, in url.
2. operation level path parameter.
3. not defined.

Reply:
Okay we have an option in ClientOptions for apiVersion, so I think 
1.for case 1 this option would take effective in path parameter
2.but i am not sure with case 2 and 3, personally i would prefer to deco the option if we plan to remove relevant core policy. not sure we have a way to do that.

Reply:
In case 2 and case 3, if there's any case that the previous policy added on query api version parameter work, this would either mean the service side has been using the api version twice or means they should have a query parameter for api version, but they don't.  Both ways don't see like a standard, I probably don't think we should support. not sure if @joheredi  or @xirzec has any different opinions on this ?

Reply:
I should point out, in the issue #2148, fabric has make it clear that they have a path parameter api version and they don't want api version version appear in their query parameter, which is the same case for face ai.

Reply:
it doesn't appear that this policy is publicly accessible to customer since core-client-rest doesn't export it, but is there a way we could indicate to the client (perhaps through a client option) to not include the policy in the first place?

Reply:
create an issue to track this https://github.com/Azure/azure-sdk-for-js/issues/28204